services/get-message.yml

```diff
@@ -17,11 +17,11 @@
   result: the_message
 
 - step: assign
-  type: assign
+  call: assign
```

Comment:
Let's use `internal` to call out internal / core Ruuter functionalities. Throughought developments.

So instead of `call: assign` let's use `internal: assign` instead.

Reply:
In the original sample there was neither and we managed to do it without them. Is it still necessary?